sheets/README.md

```diff
@@ -0,0 +1,104 @@
+# xPro Google Sheets
+
+**SECTIONS**
+1. [What Is This?](#what-is-this)
+1. [How It Works](#how-it-works)
+1. [Types of Sheets](#types-of-sheets)
+1. [Sheets Management](#sheets-management)
+1. [Development Setup](#development-setup)
+
+## What Is This?
+
+We've had a few customer service-related use cases that we have needed to support. One option in terms of implementation
+was to create several custom web UIs in xPro and give user access to all who needed it (CS representatives, finance 
+team members, etc.) This had the potential to become a persistent headache for developers, so we decided to support 
+these use cases through UIs that are very familiar to most users regardless of technical expertise: Google Sheets and
```

Comment:
```suggestion
Rather than support web UIs for internal interfaces, we chose to support Customer Support and Finance tasks through UIs that are very already familiar to them: Google Sheets and
```